### ISSUE TITLE
Issues getting https working

### ISSUE THREAD

**Issue by fstakem** [sameersbn/docker-gitlab#1435]
I am still having issues getting https working. I am using docker-compose. I changed the following:

- Line 30 to "10443:443"
- Line 50 to GITLAB_HTTPS=true
- Line 51 to SSL_SELF_SIGNED=true
- Line 54 to GITLAB_PORT=10443

I also followed the instructions to add the three random values to the docker-compose.yml. And finally I followed the instructions to create the key/cert/dh files and put them in the correct directory of the volume.

Using http works fine without the https changes, but I cannot for the life of me get https to work. I tried just about every combination of the settings above. I get the following error below. I tried to look through the gitlab logs but didn't see much. I am not sure where to debug next or what I missed.

Error code: SSL_ERROR_RX_RECORD_TOO_LONG

**Comment by iamchanghyunpark** [comment]
Yes, setting up a new gitlab docker instance, and I'm getting the same issue. 
I execed into thje container and checked that nginx was listening on the 80 port not the 443 port even though I set GITLAB_HTTPS to true

**Comment by edtriccorp** [comment]
I have been having the same issue on 10.3.6 but I am using a certificate from Letsencrypt instead of the self signed. I also cannot get SMTP email with SSL to work. HTTP and port 80 email both work however. I did open up port 10443 on my router as well.
